src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionOption/SelectionOption.styles.ts

```diff
@@ -30,7 +30,7 @@ export const StyledItemOption = styled(StyledItemOptionContainer, shouldForwardP
 
 export const StyledSvgWrapper = styled(StyledFlexTopCenter)`
   svg {
-    stroke: ${variables.palette.on_surface_variant};
+    fill: ${variables.palette.on_surface_variant};
```

Comment:
Have you also checked 'checkbox-multiple-filled' for multiple selection with this approach?